zallet/src/components/json_rpc/methods/z_get_total_balance.rs

```diff
@@ -17,23 +18,13 @@ pub(crate) type ResultType = TotalBalance;
 #[derive(Clone, Debug, Serialize, Documented, JsonSchema)]
 pub(crate) struct TotalBalance {
     /// The total value of unspent transparent outputs, in ZEC
-    transparent: f64,
+    transparent: String,
```

Comment:
Why are these not `JsonZec`?

Reply:
They were implemented as JSON strings containing numeric values, which is not the same thing as JSON numbers. (Specifically, they used `FormatMoney()` instead of `ValueFromAmount()` in `zcashd`.)